UI/Selenium/Steps/ConsultationRoomSteps.cs

```diff
@@ -110,5 +110,75 @@ public void ThenAllParticipantsLeaveConsultationRoom()
+                Driver.FindElement(ParticipantWaitingRoomPage.ParticipantDetails($"{participant.Name.FirstName} {participant.Name.LastName}")).Displayed.Should().BeTrue();
+
+                ExtensionMethods.FindElementWithWait(Driver, ParticipantWaitingRoomPage.ParticipantDetails(_hearing.Case.CaseNumber), _scenarioContext, TimeSpan.FromSeconds(Config.DefaultElementWait)).Displayed.Should().BeTrue();
+                Driver.FindElement(ParticipantWaitingRoomPage.ChooseCameraAndMicButton).Displayed.Should().BeTrue();
+                Driver.FindElement(ParticipantWaitingRoomPage.JoinPrivateMeetingButton).Displayed.Should().BeTrue();
+            }
+        }
+
+        [When(@"1st participant start a private meeting and selects 2nd participant")]
+        public void When1stParticipantStartAPrivateMeetingAndSelects2ndParticipant()
+        {
+            var participants = _hearing.Participant.Where(p => !p.Id.ToLower().Contains("judge")).ToList();
+            var firstParticipant = participants[0];
+            var firstParticipantKey = $"{firstParticipant.Id}#{firstParticipant.Party.Name}-{firstParticipant.Role.Name}";
+            Driver = GetDriver(firstParticipantKey, _scenarioContext);
+            _scenarioContext["driver"] = Driver;
+
+            var secondParticipant = participants[1];
+            var secondParticipantKey = $"{secondParticipant.Id}#{secondParticipant.Party.Name}-{secondParticipant.Role.Name}";
+
+            ExtensionMethods.FindElementWithWait(Driver, ParticipantWaitingRoomPage.StartPrivateMeetingButton, _scenarioContext, TimeSpan.FromSeconds(Config.DefaultElementWait)).Click();
+            var inputElement = Driver.FindElement(By.TagName("input"));
+
+            Driver.SwitchTo().ActiveElement();
+            var modelDialog = Driver.FindElement(ParticipantWaitingRoomPage.PrivateMeetingModal);
+            var checkbox = modelDialog.FindElement(ParticipantWaitingRoomPage.JointPrivateMeetingCheckbox(secondParticipant.Name.FirstName));
+            checkbox.Click();
+
+            modelDialog.FindElement(ParticipantWaitingRoomPage.ContinueJoiningPrivateMeetingButton).Click();
+
+            Driver = GetDriver(secondParticipantKey, _scenarioContext);
+            _scenarioContext["driver"] = Driver;
+
+            WebDriverWait wait = new WebDriverWait(Driver, TimeSpan.FromSeconds(Config.DefaultElementWait));
+            wait.Until(ExpectedConditions.ElementToBeClickable(ParticipantWaitingRoomPage.AcceptConsultationButton));
+
+            ExtensionMethods.FindElementWithWait(Driver, ParticipantWaitingRoomPage.AcceptConsultationButton, _scenarioContext).Click();
+        }
+
+        [Then(@"2nd participant is in the private consultation room")]
+        public void Then2ndParticipantIsInThePrivateConsultationRoom()
+        {
+            var participants = _hearing.Participant.Where(p => !p.Id.ToLower().Contains("judge")).ToList();
```

Comment:
See the earlier comment.

Reply:
Used "party" to identify non-staff participants